implementation.md

```diff
@@ -1554,3 +1554,268 @@ In the `Video` and `Audio` objects, sellers must only include one form of floor
+
+If an arrangement exists to put a matched DSP ID in <code>buyeruid</code>, that would be done as follows:
+
+<b>NOTE:</b> This method should be used ONLY in cases where a previous arrangement has been made directly between the sell-side and the DSP to express a matched ID in <code>buyeruid</code>. If no agreement is in place, the ids in question should be sent in the EIDS array. 
+
+```
+{
+"user": {
+    "buyeruid": "3d89748d-74bb-44f1-9908-298090dc2944",
+    "eids": [
+        {
+            "matcher": "graphvendor.com",
+            "inserter": "pub.com",
+            "source": "sspdomain.com”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a"
+		    "atype": "1"
+                }
+            ]
+        },
+        {
+            "inserter": "sspdomain.com",
+            "source": "dspdomain.com”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "3d89748d-74bb-44f1-9908-298090dc2944"
+                }
+            ]
+        }
+    ]
+}
+}
+```
+note that <code>mm</code>:3 is retained in this example, because the match is built on top of an ID that was itself based on cross-domain authentication (e.g. via hashed email)
+
+#### 7.12.4.3 Derrived IDs Across Multiple Properties or Devices
+##### Example 5: SSP matches their own ID to an email-based hash to provide a universal ID to the DSP in the buyeruid field (based on prior arrangement to put it in buyeruid)
+```
+{
+"user": {
+    "buyeruid": "fac13741-0648-436a-88cf-aceafdf45c9a",
+    "eids": [
+        {
+            "source": "sspdomain.com”,
+            "mm": 1,
+            "uids": [
+                {
+                    "id": "3d89748d-74bb-44f1-9908-298090dc2944"
+		    "atype": "3"
+                }
+            ]
+        },
+        {
+            "matcher": "idvendor.com",
+	    "inserter": "sspdomain.com",
+            "source": "universalid.com”,
+            "mm": 4,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a"
+                }
+            ]
+        }
+    ]
+}
+}
+```
+##### Example 6: On a CTV request, the publisher also provides the IFA and an alternative ID of an Apple device within that same household, based on authenticated logins across apps
+
+```
+{
+"user": {
+    "eids": [
+        {
+	    "matcher": "app-pub.com",
+	    "inserter": "app-pub.com",
+	    "source": "apple.com”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a"
+		    "atype": "2"
+                }
+            ]
+        },
+        {
+            "matcher": "app-pub.com",
+	    "inserter": "app-pub.com",
+            "source": "alt-id.org”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "e7f8c5f2-6b4e-4565-8b0a-53d8b730143e"
```

Comment:
missing "atype":3

Reply:
Though I suppose depending on the ID (if it's partitioned by app bundle), could technically be "atype":2, as well.

Reply:
added as 3, but i'll defer to you and @patmmccann for all the atype values